centrifuge/src/main/java/io/github/centrifugal/centrifuge/Subscription.java

```diff
@@ -12,6 +13,12 @@
 
     private Client client;
     private String channel;
+    private long lastOffset;
+    private boolean recover;
+    private long subscribedAt = 0;
+    private String lastEpoch;
+    private int lastSeq;
```

Comment:
There is no need to use Seq and Gen at all because we work with Offset only here, or am I missing something?

Reply:
@FZambia sorry, I'm missing reduce it from Subscription class whether it is not used. to day i will removed it.